packages/fxa-settings/src/pages/Signin/SigninTokenCode/mocks.tsx

```diff
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import React from 'react';
```

Comment:
This is to resolve an error in storybook - this story was not loading because of this missing import.